x/escrow/keeper/keeper.go

```diff
@@ -521,7 +582,7 @@ func accountSettleFullblocks(
 	sdk.Coin,
 ) {
 
-	numFullBlocks := account.Balance.Amount.Quo(blockRate.Amount)
+	numFullBlocks := account.Balance.Amount.Add(account.Funds.Amount).Quo(blockRate.Amount)
```

Comment:
can we do `account.Balance.Add(account.Funds)` here?  Maybe we should make a `TotalBalance()` method on `Account`.

Reply:
Done